doc/nrf/links.txt

```diff
@@ -91,7 +91,6 @@
 .. _`SMP over Bluetooth`: https://github.com/apache/mynewt-mcumgr/blob/master/transport/smp-bluetooth.md
 
 .. _`Android CHIPTool`: https://github.com/nrfconnect/sdk-connectedhomeip/tree/ee7ed2a/src/android/CHIPTool
-.. _`Commissioning nRF Connect Accessory using Android CHIPTool`: https://github.com/nrfconnect/sdk-connectedhomeip/blob/ee7ed2a/docs/guides/nrfconnect_android_commissioning.md
 .. _`Working with Python Controller`: https://github.com/nrfconnect/sdk-connectedhomeip/blob/ee7ed2a/docs/guides/python_chip_controller_building.md
```

Comment:
may also be removed/updated?

Reply:
Yes, I want to test two links first, as I cannot build locally yet (hasn't had time and last week had errors all the time).